README.md

```diff
@@ -7,6 +7,12 @@ Opn Payments provides a set of APIs that help merchants of any size accept payme
 
 **Omise WooCommerce** is our official plugin providing support for processing payments on WooCommerce through Opn Payments.
 
+## Security Warning
+
+**Please do NOT use versions less than 4.8, as they are outdated and have security vulnerabilities..**
```

Comment:
@muthuswamyopn I think it would be better to include sub-string `Omise WooCommerce` before `4.8` so that readers would be clear that we are talking about plugin and not the WordPress or WooCommerce.